README.md

```diff
@@ -141,22 +143,22 @@ Esri welcomes contributions from anyone and everyone. Please see our [guidelines
 
 In order to make use of the ArcGIS Online World Geocoding Service in your web application:
 
-1. Please sign up for an [ArcGIS for Developers account](https://developers.arcgis.com/en/plans) or purchase an [ArcGIS Online Organizational Subscription](http://www.arcgis.com/features/plans/pricing.html).
+1. Please sign up for an [ArcGIS Developer account](https://developers.arcgis.com) or purchase an [ArcGIS Online Organizational Subscription](http://www.arcgis.com/features/plans/pricing.html).
 2. Ensure that `Powered by`[`Esri`](http://esri.com) is displayed in the map attribution.
 
 ## Cost
 
-1. User search for individual locations within web applications is defined by Esri as _Geosearch_ and it is free. [Credits](https://developers.arcgis.com/credits/) are only consumed when you store results permanently.
-2. To store geocoding results, pass `forStorage: true` and a valid access token (see [#207](https://github.com/Esri/esri-leaflet-geocoder/issues/207#issuecomment-418763785)).
-3. You are not allowed to generate revenue while on a free ArcGIS Developer plan.
-4. If your application generates more than 1 million searches in a month, please [contact us](http://www.esri.com/about-esri/contact).
+ArcGIS geocode results have two different costs depending if the results you receive will be used only during your app session or stored permanently. Refer to the [ArcGIS Developer pricing page](https://developers.arcgis.com/pricing/) for more information regarding costs. See [Search for addresses and places](https://developers.arcgis.com/documentation/mapping-apis-and-services/search/) for more information on the available geocoding services.
```

Comment:
```suggestion
Some geocoding is free with the geocoding service, but it depends on the type of account you have and if you are not storing or storing geocoding results. To determine if there is cost, if you have an ArcGIS Developer account, go to [Pricing](https://developers.arcgis.com/pricing/). If you have an ArcGIS Online account, go to [Credits by Capability](https://doc.arcgis.com/en/arcgis-online/administer/credits.htm#GUID-D309A4D0-43CD-4E58-A6DF-012A82A6D794).
```

Reply:
need to fix the punctuation on this suggestion